examples/using-file-bind/main.go

```diff
@@ -39,13 +39,12 @@ func UploadHandler(c *gofr.Context) (interface{}, error) {
 		return nil, err
 	}
 
-	// create local copies of the zipped files in tmp folder
-	err = d.Compressed.CreateLocalCopies("tmp")
-	if err != nil {
-		return nil, err
-	}
-
-	defer os.RemoveAll("tmp")
+	// Defer removing the temporary folder after the function exits
+	defer func() {
+		if err := os.RemoveAll("tmp"); err != nil {
+			fmt.Println("Error removing tmp folder:", err)
+		}
```

Comment:
Instead of fmt.Println, we should be using `c.Logger`

Reply:
@RupeshHacker using `c.Logger()` is a syntactical error, please use `Errorf() `or `Error()` methods provided by c.Logger